Cargo.toml

```diff
@@ -5,7 +5,14 @@ members = ["hotham", "hotham-simulator", "examples/simple-scene", "examples/crab
 [profile.dev.package.rapier3d]
 opt-level = 3
 
-[profile.dev.package.base64]
+[profile.dev.package.parry3d]
+opt-level = 3
+
+# Make performance critical packages compile with optimizations
+[profile.dev.package.nalgebra]
+opt-level = 3
+
+[profile.dev.package."base64"]
```

Comment:
Why are there quotes around base64 now and not before?

Reply:
Ah, somebody had suggested this as a fix for getting cargo to optimise at the right level. Didn't do anything, so I'll revert that!